packages/core/src/indexing/migrations/1-create-model-table.ts

```diff
@@ -38,44 +38,48 @@ export type TableIndices = {
   indices: Array<TableIndex>
 }
 
+export function indexNameFromTableName(tableName: string): string {
+  return `idx_${tableName.substring(tableName.length - 10)}`
+}
+
 export function indices(tableName: string): TableIndices {
   // create unique index name less than 64 chars that are still capable of being referenced to MID table.
```

Comment:
Nitpick - can you move the comment above the newly created helper function so we have the explanation captured at the right place.